examples/seismic/tti/tti_example3D.py

```diff
@@ -57,18 +57,19 @@ def setup(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     rec = Receiver(name='rec', ntime=nt, coordinates=receiver_coords)
 
     return AnisotropicWaveSolver(model, source=src, time_order=time_order,
-                                 space_order=space_order, receiver=rec, dse=dse)
+                                 space_order=space_order, receiver=rec, dse=dse, dle=dle,
+                                 compiler=compiler)
```

Comment:
OK, so the *right* fix is to drop ``compiler``, even from ``benchmark.py``, because that's not how it is set now